src/wms/src/test/java/org/geoserver/wms/wms_1_1_1/LayerWorkspaceTest.java

```diff
@@ -47,10 +46,12 @@ public void testLayerOrderGetCapabilities() throws Exception {
         Document doc = getAsDOM("/wms?service=WMS&request=getCapabilities&version=1.1.1", true);
         List<String> originalList = layerNameList(doc);
         assertFalse(originalList.isEmpty());
-        List<String> names =
-                originalList.stream().map(x -> removeLayerPrefix(x)).collect(Collectors.toList());
-        List<String> orderedNames = names.stream().sorted().collect(Collectors.toList());
-        assertTrue(orderedNames.equals(names));
+        int prevIndex = 0;
```

Comment:
Same as above, without setting any index the previous test checks the desired behavior and should not be modified.